src/main/java/org/broadinstitute/hellbender/utils/recalibration/covariates/ContextCovariate.java

```diff
@@ -188,7 +188,7 @@ private static IntList contextWith(final byte[] bases, final int contextSize, fi
             currentKey = 0;
             currentNPenalty = contextSize - 1;
             int offset = newBaseOffset;
-            while (bases[currentNPenalty] != 'N') {
+            while (BaseUtils.simpleBaseToBaseIndex(bases[currentNPenalty]) != -1) {
```

Comment:
Can you fix the inaccurate comment above that `if the first key was -1 then there was an N in the context`?

Also, `BaseUtils.simpleBaseToBaseIndex()` will return 0 for '*' -- is this something we need to care about here?

Reply:
for some reason we assume that * == A in this context...not sure how that helps....but if someone has a '*' in their non-empty read, God help them....